### DISCUSSION TITLE
Whats the difference beetween calendar.work_holiday and calendar.work_vacation_jason in script work_schedule.yaml ?

### DISCUSSION THREAD

**Discussion by Headkillah** [jazzyisj/home-assistant-config#86] [General]
Hello @jazzyisj. How are you?

I´m struggeling a little bit with some parts of "work_schedule.yaml" in templates. What´s the difference beetween calendar.work_holiday and calendar.work_vacation_jason for the "Work Today" and "Work Tomorrow" Binary Sensors?

Thx for your answer :)

Greetz Headkillah

**Comment by jazzyisj** [comment]
calendar.work_holiday are individual days that are company holidays (including federal, provincial holidays) and individual personal days I take off of work.   If I book a day off at the last minute or production is cancelled last minute I can just toggle the work today off or work tomorrow off boolean instead of putting it in the calendar.  I also have work today and work tomorrow override booleans I can toggle when I'm called into work unexpectedly.

 calendar.work_vacation_jason are booked vacation weeks (I'm required to take my paid vacation in 1 week blocks).

They do the same thing.  I could have done them all on one calendar if I was into keeping things simple. :-)

**Reply by Headkillah** [reply]
Ahhhh now a little bit more gets clearer :) Because i´m living in germany here are the rules for taking vacation are different. So i don´t need the override booleans because i have a 5-days week and 2 days weekend👍But still i have 2 calendars, 1 for my vacation.

Thx @jazzyisj for your answer, i will come back to ask more :D
